### PR TITLE
player selfsame has entered the game

### PR DESCRIPTION
**Pull request by selfsame**

### CODE REVIEW

File: players/001-selfsame [hunk 1]
```diff
@@ -0,0 +1 @@
+12
```
Comment (audiodude) [inline players/001-selfsame]: Uh nice try. Your initial score is 0.

**Review by audiodude — CHANGES_REQUESTED** [review]

File: players/001-selfsame (hunk 1 above)
Comment (selfsame) [reply players/001-selfsame]: By what rule?

Comment (selfsame) [reply players/001-selfsame]: ah i see it https://github.com/audiodude/tiny-nomic/blob/21429fde4eb8455e72743e2946d37aa889677082/mutable_rules/201_Turn-alternation.md

**Review by audiodude — APPROVED** [review]
I really appreciate the spirit though!

**Pull request merged**